dev/io.openliberty.checkpoint_fat/test-applications/mpapp1/src/mpapp1/ApplicationScopedOnCheckpointBeanWithConfigObject.java

```diff
@@ -22,13 +22,14 @@
 @ApplicationScoped
 public class ApplicationScopedOnCheckpointBeanWithConfigObject {
 
+    @Inject
+    Config config;
+
     public void observeInit(@Observes @Initialized(ApplicationScoped.class) Object event) {
         System.out.println(getClass() + ": " + "Initializing application context");
+        check("defaultValue");
     }
```

Comment:
I don't think anything checks that this `check()` call doesn't throw an error.

If it did, I think it would fail in `configObjectAppScopeEnvValueChangeTest`

Reply:
It might be that you're not fussed about actually checking what the value is at this point. In that case you could read the value from config manually rather than calling `check()`.